src/mitochondria/__test__/auth.test.ts

```diff
@@ -40,15 +40,11 @@ describe('Authentication/Registration', () => {
       loginPassword
     );
     expect(registerResp).toEqual({
-      access,
-      refresh,
-      user: {
-        companies: [],
-        email: loginEmail,
-        first_name: loginFirstName,
-        id: 0,
-        last_name: loginLastName,
-      },
+      companies: [],
+      email: loginEmail,
+      first_name: loginFirstName,
+      id: 0,
+      last_name: loginLastName,
```

Comment:
It's because the API.register strips away the tokens as they're only stored in localStorage. This is now the intended behaviour, as there is no reason to return the tokens anywhere. :)